src/itensor.jl

```diff
@@ -358,6 +358,11 @@ end
 
 const Indices = Union{IndexSet,Tuple{Vararg{Index}}}
 
+"""
+randomITensor([S,] inds)
```

Comment:
Please add 4 spaces at the beginning of the line, so the Markdown processor will display this line as code instead of text. Minor thing but helpful & conventional.

Reply:
Makes sense. Done.